megamek/src/megamek/client/ui/swing/MiniMap.java

```diff
@@ -1777,25 +1339,25 @@ public void gamePhaseChange(GamePhaseChangeEvent e) {
             if (GUIPreferences.getInstance().getGameSummaryMiniMap() && ((e.getOldPhase() == Phase.PHASE_DEPLOYMENT)
                     || (e.getOldPhase() == Phase.PHASE_MOVEMENT) || (e.getOldPhase() == Phase.PHASE_TARGETING)
                     || (e.getOldPhase() == Phase.PHASE_FIRING) || (e.getOldPhase() == Phase.PHASE_PHYSICAL))) {
-                File dir = new File(Configuration.gameSummaryImagesMMDir(), m_game.getUUIDString());
+                File dir = new File(Configuration.gameSummaryImagesMMDir(), game.getUUIDString());
                 if (!dir.exists()) {
                     dir.mkdirs();
                 }
-				File imgFile = new File(dir, "round_" + m_game.getRoundCount() + "_" + e.getOldPhase().ordinal() + "_"
+				File imgFile = new File(dir, "round_" + game.getRoundCount() + "_" + e.getOldPhase().ordinal() + "_"
 						+ IGame.Phase.getDisplayableName(e.getOldPhase()) + ".png");
                 try {
-                    ImageIO.write(m_mapImage, "png", imgFile);
+                    ImageIO.write(getMinimapImage(game, bv, 4), "png", imgFile);
```

Comment:
Can we make a constant out of `4` so that we have a nice name for it (besides: doesn't show controls).

Reply:
> (besides: doesn't show controls)
Sorry I don't get what you mean with this.

Reply:
Ah, like why `4`, is it because it doesn't show controls on the minimap?

Reply:
Oh, the controls aren't shown because the minimap images are created without a dialog and this
```
        if (dialog != null) {
            drawButtons(g);
            repaint();
        }
```
keeps the buttons from being painted. 
The 4 value is just some middle value; there's no ideal size like for the actual map (where we scale the images, so no zoom other than 1 makes real sense). On the biggest zoom, the biggest maps create minimap images of about 2500 x 4000. Compared to the boardview images that's still small. I just checked: The 160x170 maps make 50MB board images of 8000x10000 (and it takes 6 seconds to write that image). Hmmm. I could set this to full minimap zoom. I'm undecided.